iso690-author-date-en.csl

```diff
@@ -118,7 +118,7 @@
       <if type="book thesis map motion_picture song manuscript" match="any">
         <choose>
           <if variable="author editor translator" match="any">
-            <text variable="title" font-style="italic"/>
+            <text variable="title" font-style="italic" suffix=". "/>
```

Comment:
Are you making the changes to all the other ISO-690 styles on purpose? (the PR title only mentions HEG). And if so, what's the idea of adding all the periods in the title suffix here? The title macro has period in the suffix every time it's called in the bibliography, so this shouldn't be necessary (and makes the style harder to maintain)

Reply:
Thanks @adam3smith. You are right. I removed these periods in the title macro.